api/cas-server-core-api-events/src/main/java/org/apereo/cas/support/events/dao/CasEvent.java

```diff
@@ -37,27 +40,33 @@
 @ToString
 @Getter
 @Setter
+@AllArgsConstructor
 public class CasEvent {
 
     @org.springframework.data.annotation.Id
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
     @GenericGenerator(name = "native", strategy = "native")
+    @JsonProperty("id")
     private long id = -1;
 
+    @JsonProperty("type")
     @Column(nullable = false)
     private String type;
 
+    @JsonProperty("principalId")
     @Column(nullable = false)
     private String principalId;
 
+    @JsonProperty("creationTime")
     @Column(nullable = false)
     private String creationTime;
 
+    @JsonProperty("properties")
     @ElementCollection
     @MapKeyColumn(name = "name")
     @Column(name = "value")
-    @CollectionTable(name = "events_properties", joinColumns = @JoinColumn(name = "id"))
```

Comment:
Why this change?

Reply:
because I changed `getId()` to avoid collision with the `id` field.